tests/integration/llm/prepare.py

```diff
@@ -1145,6 +1157,24 @@ def build_correctness_model(model):
     write_model_artifacts(options)
 
 
+def build_text_embedding_model(model):
+    if model not in text_embedding_model_list:
+        raise ValueError(
+            f"{model} is not one of the supporting handler {list(onnx_list.keys())}"
+        )
+    options = text_embedding_model_list[model]
+    options["engine"] = "Rust"
+    options["option.task"] = "text_embedding"
+    options["option.mapLocation"] = "true"
```

Comment:
We don't need mapLocation for Rust engine

Reply:
Removed.